yamllint/rules/__init__.py

```diff
@@ -14,6 +14,8 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import yamllint.plugins
+
```

Comment:
This space is not needed I think, `yamllint.*` import should be grouped.

Reply:
Please let me clarify that your thought. Do you think that it should be something like:

```python
from yamllint import (
    plugins,
    rules.braces,
    ...
)
```
or anything else?

I fully agree with you about that the empty line 18 is not needed and pushed that change already (commit ad1f292) but I'm not sure to group those (plugin and rules.*) imports is good thing from your maintainer's point of view (I think of the cases that addition and/or removal of standard rules).